servicetests/tests/ShadowUpdate/shadow_update.py

```diff
@@ -0,0 +1,480 @@
+            request=iotshadow.UpdateNamedShadowSubscriptionRequest(shadow_name=named_shadow, thing_name=shadow_thing_name),
+            qos=mqtt_qos,
+            callback=on_update_shadow_rejected)
+
+        # Wait for subscriptions to succeed
+        update_accepted_subscribed_future.result()
+        update_rejected_subscribed_future.result()
+
+        print("Subscribing to Get responses...")
+        get_accepted_subscribed_future, _ = shadow_client.subscribe_to_get_named_shadow_accepted(
+            request=iotshadow.GetNamedShadowSubscriptionRequest(shadow_name=named_shadow,thing_name=shadow_thing_name),
+            qos=mqtt_qos,
+            callback=on_get_shadow_accepted)
+
+        get_rejected_subscribed_future, _ = shadow_client.subscribe_to_get_named_shadow_rejected(
+            request=iotshadow.GetNamedShadowSubscriptionRequest(shadow_name=named_shadow, thing_name=shadow_thing_name),
+            qos=mqtt_qos,
+            callback=on_get_shadow_rejected)
+
+        # Wait for subscriptions to succeed
+        get_accepted_subscribed_future.result()
+        get_rejected_subscribed_future.result()
+
+        # Issue request for shadow's current state.
+        # The response will be received by the on_get_accepted() callback
+        print("Requesting current shadow state...")
+
+        with locked_data.lock:
+            # use a unique token so we can correlate this "request" message to
+            # any "response" messages received on the /accepted and /rejected topics
+            token = str(uuid4())
+
+            publish_get_future = shadow_client.publish_get_named_shadow(
+                request=iotshadow.GetNamedShadowRequest(shadow_name=named_shadow, thing_name=shadow_thing_name, client_token=token),
+                qos=mqtt_qos)
+
+            locked_data.request_tokens.add(token)
+
+        # Ensure that publish succeeds
+        publish_get_future.result()
+
+        subscribe_future, _ = shadow_client.subscribe_to_named_shadow_updated_events(
+            request=iotshadow.NamedShadowUpdatedSubscriptionRequest(shadow_name=named_shadow, thing_name=shadow_thing_name),
+            qos=mqtt_qos,
+            callback=update_event_received)
+        subscribe_future.result()
+
+        state=iotshadow.ShadowState(
+            reported={shadow_property: cmdData.input_shadow_value},
+            desired={shadow_property: cmdData.input_shadow_value},
+            token=token)
+
+        update_thing_update_future = shadow_client.publish_update_named_shadow(request = iotshadow.UpdateNamedShadowRequest
+                (shadow_name = named_shadow, thing_name = shadow_thing_name, state=state), qos=mqtt_qos)
+
+        # Wait for subscriptions to succeed
+
+        # Launch thread to handle user input.
+        # A "daemon" thread won't prevent the program from shutting down.
+        print("Launching thread to read user input...")
```

Comment:
Seeing this message in CI might be confusing. It's better to clean this test from log messages like this one.